docs/develop/dotnet/index.mdx

```diff
@@ -14,7 +14,22 @@ tags:
 
 <!-- .NET SDK feature guidance landing page-->
 
-Build [Temporal Applications](/temporal#temporal-application) with the .Net SDK.
+![.NET SDK Banner](/img/banner-dotnet-temporal.png)
+
+:::info .NET SPECIFIC RESOURCES
+Build [Temporal Applications](/temporal#temporal-application) with the .NET SDK.
+
+**Temporal .NET Technical Resources:**
+- [.NET API Documentation](https://docs.temporal.io/docs/dotnet)
```

Comment:
https://docs.temporal.io/docs/dotnet doesn't resolve. Should this be https://dotnet.temporal.io/api/ ?

Reply:
Yup! Thank you! 🙌